docs/output.md

```diff
@@ -2,58 +2,265 @@
 
 ## Introduction
 
-This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+This document describes the output produced by the pipeline based on public data.
 
-The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
+The directories listed below are created in the `results/` directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
+### Dataset
+
+RNAseq data has been taken from GM12878 GIAB sample SRA Accession [SRR5665260](https://www.ncbi.nlm.nih.gov/sra/?term=SRR5665260). The dataset has 38.6 million PE reads of read length 2x151 bp from NextSeq 500 sequencing platform.
+
+### Samplesheet
+
+A sample sheet has been prepared in the following way to set the FASTQ files to run the analysis.
+
+```console
+sample,fastq_1,fastq_2,strandedness
+GM12878,/data/GM12878/SRR5665260_1.fastq.gz,/data/GM12878/SRR5665260_2.fastq.gz,reverse
+```
+
+### Execution
+
+The pipeline has been executed with the following command.
+
+```console
+nextflow run nf-core/rnavar -profile munin,docker --input samplesheet.csv --genome GRCh38 --read_length 151 --annotate_tools merge
+```
+
+It used an institutional profile `munin` from [nf-core/configs](https://github.com/nf-core/configs/blob/master/conf/pipeline/rnavar/munin.config)
```

Comment:
I'm sure we need to have the example here tied to `munin`

Reply:
@praveenraj2018 you haven't replied to this comment

Reply:
@maxulysse I did not understand this comment. What example should be given here?

Reply:
I am not sure about this, so leaving it for now.!

Reply:
Agree, munin is a bit specific IMO for a general explanation. Worried people will c&p this to run their own analyses (especially new ones to the party) confused why it won't work on their system. How about `munin` -> `<initutional_config>` ? inistutional config should be searchable on the website

Reply:
Just changed to test profile. So, just in case if people c&p, the pipeline will work for the test datasets. Also added a note on institutional profile and link to it.